terraform/templates/iam_apigateway.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "apigateway:CreateKey",
+                "apigateway:ListKey"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
```

Comment:
This seems out of place. Does it belong in this PR?

Reply:
good point, idk where this came from